src/nvim/eval/funcs.c

```diff
@@ -5977,7 +5979,6 @@ static void f_localtime(typval_T *argvars, typval_T *rettv, FunPtr fptr)
   rettv->vval.v_number = (varnumber_T)time(NULL);
 }
 
-
 static void get_maparg(typval_T *argvars, typval_T *rettv, int exact)
```

Comment:
I originally wanted to move this function to `ex_map.c`, but this function is only used by functions in `eval/funcs.c`.